spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafka.java

```diff
@@ -80,6 +80,11 @@
 	 * @return passed into {@code kafka.utils.TestUtils.createBrokerConfig()}.
 	 */
 	boolean controlledShutdown() default false;
+	
+	/**
+	 * @return ports for brokers.
```

Comment:
Please, add `@since 2.2.4` into this Javadoc.
Or do we switch to the `2.3.0` already, @garyrussell ?

Reply:
No; this can go into 2.2.4.

Reply:
I also want to see in this Javadoc an explanation that number of ports provided must be equal to the `count` value.